apps/frontend-manage/src/components/activities/overview/ActivityDetailsModal.tsx

```diff
@@ -2,22 +2,43 @@ import { useQuery } from '@apollo/client'
 import { faClock } from '@fortawesome/free-regular-svg-icons'
 import {
   faArrowUpRightFromSquare,
+  faBookOpen,
+  faCheckCircle,
   faExclamationTriangle,
   faUserGroup,
 } from '@fortawesome/free-solid-svg-icons'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import {
   ActivityInfo,
+  ActivityInfoElement,
   ActivityType,
+  ElementInstance,
   GetActivityDetailsDocument,
   GetOutdatedElementInstancesDocument,
   PublicationStatus,
 } from '@klicker-uzh/graphql/dist/ops'
-import { H4, Modal, UserNotification } from '@uzh-bf/design-system'
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+  Modal,
+  RadioGroup,
+  RadioGroupItem,
+  ShadcnTable,
+  ShadcnTableBody,
+  ShadcnTableCell,
+  ShadcnTableFooter,
+  ShadcnTableHead,
+  ShadcnTableHeader,
+  ShadcnTableRow,
+  UserNotification,
+} from '@uzh-bf/design-system'
 import { useTranslations } from 'next-intl'
 import Link from 'next/link'
-import { useMemo } from 'react'
+import { useMemo, useState } from 'react'
 import { twMerge } from 'tailwind-merge'
+import StudentElementPreviewActivityDetails from '~/components/questions/manipulation/StudentElementPreviewActivityDetails'
```

Comment:
for consistency with conventions across project :)

```suggestion
import StudentElementPreviewActivityDetails from '../../questions/manipulation/StudentElementPreviewActivityDetails'
```